### PR TITLE
Fix flatpak discord's RPC not working with flatpak'd headset

### PR DESCRIPTION
**Pull request by ghost**
Fix issue where discord flatpak and headset flatpak could not communicate with discords RPC.

### CODE REVIEW

File: co.headsetapp.headset.yml [hunk 1]
```diff
@@ -32,6 +33,9 @@ modules:
         dest-filename: headset.sh
         commands:
           - zypak-wrapper /app/headset/headset "$@"
+          - for i in {0..9}; do \
+          -    test -S $XDG_RUNTIME_DIR/discord-ipc-$i || ln -sf {app/com.discordapp.Discord,$XDG_RUNTIME_DIR}/discord-ipc-$i; \
+          -  done
```
Comment (fcastilloec) [inline co.headsetapp.headset.yml:37]: These lines should be placed before we run Headset, before `zypak-wrapper /app/headset/headset "$@"`

**Review by fcastilloec — CHANGES_REQUESTED** [review]

File: co.headsetapp.headset.yml (hunk 1 above)
Comment (ghost) [reply co.headsetapp.headset.yml:37]: This is true, completely forgot to swap them round again xD

**Comment by fcastilloec** [comment]
> Build [111544 successful](https://buildbot.flathub.org/#/builders/32/builds/111544) To test this build, install it from the testing repository:
> 
> ```
> flatpak install --user https://dl.flathub.org/build-repo/109156/co.headsetapp.headset.flatpakref
> ```

@pinkuuu1 I encourage you to test the app with your changes by installing it with the above command. Let me know if everything is working with Discord as expected.

**Comment by ghost** [comment]
> > Build [111544 successful](https://buildbot.flathub.org/#/builders/32/builds/111544) To test this build, install it from the testing repository:
> > ```
> > flatpak install --user https://dl.flathub.org/build-repo/109156/co.headsetapp.headset.flatpakref
> > ```
> 
> @pinkuuu1 I encourage you to test the app with your changes by installing it with the above command. Let me know if everything is working with Discord as expected.

Works fine for me!

**Comment by fcastilloec** [comment]
I'll see if I can also test it tonight, if I also confirm it, we can merge it tonight and you'll probably see the new version in the next few days (flathub takes a couple of days to publish builds)

**Review by fcastilloec — APPROVED** [review]

**Comment by ghost** [comment]
> I'll see if I can also test it tonight, if I also confirm it, we can merge it tonight and you'll probably see the new version in the next few days (flathub takes a couple of days to publish builds)

Awesome sounds like a plan, i'll keep this tab open in my browser 👍

**Comment by fcastilloec** [comment]
@pinkuuu1 I installed Discord on my VM and Headset from this PR build and it doesn't seem to be working for me. Did you install Discord from flatpak or another method?

**Comment by ghost** [comment]
@fcastilloec i installed discord flatpak and then headset flatpak too. discord from flathub

**Comment by fcastilloec** [comment]
I can confirm that this is working but it only works with Discord installed from flatpak. If you install Discord through any other method (usually one that is not sandboxed) it doesn't work. I'm going to merge this PR but I won't close the original issue you posted on the main repo, since I'll have to investigate how to make it work for the other cases.

**Pull request merged**